app/assets/javascripts/visualizations/highvis/baseVis.coffee.erb

```diff
@@ -36,17 +36,18 @@ $ ->
       constructor: (@canvas) ->
         @configs ?= {}
 
-      ANALYSISTYPE_TOTAL:     0
-      ANALYSISTYPE_MAX:       1
-      ANALYSISTYPE_MIN:       2
-      ANALYSISTYPE_MEAN:      3
-      ANALYSISTYPE_MEDIAN:    4
-      ANALYSISTYPE_COUNT:     5
+      ANALYSISTYPE_TOTAL:      0
+      ANALYSISTYPE_MAX:        1
+      ANALYSISTYPE_MIN:        2
+      ANALYSISTYPE_MEAN:       3
+      ANALYSISTYPE_MEAN_ERROR: 4
```

Comment:
This is not compatible with saved/default regressions. Anything saved with analysis type 4 will now have mean_error instead of median.